README.md

```diff
@@ -1,3 +1,6 @@
-# war_game
-#
-# $ ruby war.rb
+# War, The Card Game
+```
+ $ ruby war.rb
+```
+
+![Example Play](ss.png)
```

Comment:
What does this do?

Reply:
I uploaded a picture with Gameplay, and then added that picture to the README.